pages/index.js

```diff
@@ -1,69 +1,55 @@
-import Head from 'next/head'
-import Image from 'next/image'
-import styles from '../styles/Home.module.css'
+import { useState } from 'react'
+import Link from 'next/link'
+import useStudies from '@hooks/use-studies'
+import { post } from '@lib/utils'
 
 export default function Home() {
-  return (
-    <div className={styles.container}>
-      <Head>
-        <title>Create Next App</title>
-        <meta name="description" content="Generated by create next app" />
-        <link rel="icon" href="/favicon.ico" />
-      </Head>
-
-      <main className={styles.main}>
-        <h1 className={styles.title}>
-          Welcome to <a href="https://nextjs.org">Next.js!</a>
-        </h1>
-
-        <p className={styles.description}>
-          Get started by editing{' '}
-          <code className={styles.code}>pages/index.js</code>
-        </p>
-
-        <div className={styles.grid}>
-          <a href="https://nextjs.org/docs" className={styles.card}>
-            <h2>Documentation &rarr;</h2>
-            <p>Find in-depth information about Next.js features and API.</p>
-          </a>
-
-          <a href="https://nextjs.org/learn" className={styles.card}>
-            <h2>Learn &rarr;</h2>
-            <p>Learn about Next.js in an interactive course with quizzes!</p>
-          </a>
-
-          <a
-            href="https://github.com/vercel/next.js/tree/canary/examples"
-            className={styles.card}
-          >
-            <h2>Examples &rarr;</h2>
-            <p>Discover and deploy boilerplate example Next.js projects.</p>
-          </a>
-
-          <a
-            href="https://vercel.com/new?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
-            className={styles.card}
-          >
-            <h2>Deploy &rarr;</h2>
-            <p>
-              Instantly deploy your Next.js site to a public URL with Vercel.
-            </p>
-          </a>
-        </div>
-      </main>
-
-      <footer className={styles.footer}>
-        <a
-          href="https://vercel.com?utm_source=create-next-app&utm_medium=default-template&utm_campaign=create-next-app"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Powered by{' '}
-          <span className={styles.logo}>
-            <Image src="/vercel.svg" alt="Vercel Logo" width={72} height={16} />
-          </span>
-        </a>
-      </footer>
-    </div>
-  )
+  const user = 'Dr. Coan'
```

Comment:
Usernames should not have a space in them. We need to discuss DB structure more regarding what collections we need and what data should be in those collections.

Reply:
Here's my rough draft of the DB schema:

Study Schema
- Name: string 
- Admin: string
- Key: string
- Data: List<UserData>
- Song ID: int
- Profile: enum

UserData Schema
- ID: int
- HorizSliderInputs: List<Input>?
- VertSliderInputs: List<Input>?
- ButtonOneInputs: List<Input>?
- ButtonTwoInputs: List<Input>?
- ButtonThreeInputs: List<Input>?
- ToggleOneInputs: List<Input>?
- ToggleTwoInputs: List<Input>?
- ToggleThreeInputs: List<Input>?
- JoystickInputs: List<Input>?

Input Schema
- Timestamp : Time
- Value : int

Song Schema
- ID
- Audio file

Reply:
This is a good start. We can meet to finalize db structure some time soon.